pkg/ovs/openflow/ofctrl.go

```diff
@@ -0,0 +1,290 @@
+package openflow
```

Comment:
Should we name this file ofctrl_bridge (I know it also has ofTable code, but seems bridge is the major interface).

Reply:
Done.